Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
 gem 'benchmark-ips', '>= 2.0'
-
+gem 'activesupport'
```

Comment:
I don't think we should have a dependency on ActiveSupport. And if we do, it really should be locked down, otherwise the results are subject to change every time Rails issues a release of any sort.